### PR TITLE
updated 

### PR DESCRIPTION
**Pull request by thesyedsufiyan**
@gkanishk can you accept

### CODE REVIEW

**Comment by gkanishk** [comment]
Please resolve merge conflicts

**Comment by thesyedsufiyan** [comment]
@gkanishk fine?

File: content/finalist/ttjc@finalist@thesyedsufiyan/index.md [hunk 1]
```diff
@@ -2,20 +2,19 @@
...
-description: "Javascript | React | node.js | Html | Css"
-status: ''
----
+portfolio: "https://syedportfolio.netlify.app/"
+github-link: "https://github.com/theSyedsufiyan"
+linkedin-link: "https://www.linkedin.com/in/syed-sufiyans/"
+looking-for: "job/internship"
+image-link: "https://avatars0.githubusercontent.com/u/36100349?s=460&u=522f57940e95f59417f7ac6c6d27c9a84a7ae6ae&v=4"
+description: "Javascript | React | node.js | Html | Css | firebase | sass"
+
```
Comment (gkanishk) [inline content/finalist/ttjc@finalist@thesyedsufiyan/index.md:11]: You have changed the naming convention kindly have a look also I added a status tag please add that leave it blank by default 
```
status: ""
```
image

**Review by gkanishk — COMMENTED** [review]
kindly go through the comments

File: content/finalist/ttjc@finalist@thesyedsufiyan/index.md (hunk 1 above)
Comment (thesyedsufiyan) [reply content/finalist/ttjc@finalist@thesyedsufiyan/index.md:11]: yes i changed portfolio name , added 2 projects 
status keep empty.

**Pull request merged**